src/presets/100early_data.asm

```diff
@@ -1,4102 +1,3562 @@
-    dl $7ED906 : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED908 : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED90A : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED90C : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED90E : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED910 : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED912 : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED914 : db $02 : dw $001F ; Events, Items, Doors
-    dl $7ED916 : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED918 : db $02 : dw $0006 ; Events, Items, Doors
-    dl $7ED91A : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED91C : db $02 : dw $0000 ; Events, Items, Doors
-    dl $7ED91E : db $02 : dw $0000 ; Events, Items, Doors
+    dw $078B, $0000  ; Elevator Index
+    dw $078D, $AB58  ; DDB
+    dw $079B, $DF45  ; MDB
+    dw $07F3, $002D  ; Music Bank
+    dw $07F5, $0006  ; Music Track
+    dw $090F, $0000  ; Screen subpixel X position
+    dw $0911, $0000  ; Screen X position in pixels
+    dw $0913, $0000  ; Screen subpixel Y position
+    dw $0915, $0000  ; Screen Y position in pixels
+    dw $0917, $0000  ; Layer 2 X position
+    dw $0919, $0000  ; Layer 2 Y position
+    dw $0921, $0000  ; BG2 X offset
+    dw $0923, $0000  ; BG2 Y offset
+    dw $093F, $0000  ; Ceres escape flag
+    dw $09A2, $0000  ; Equipped Items
+    dw $09A4, $0000  ; Collected Items
+    dw $09A6, $0000  ; Beams
+    dw $09A8, $0000  ; Beams
+    dw $09C0, $0000  ; Manual/Auto reserve tank
+    dw $09C2, $0063  ; Health
+    dw $09C4, $0063  ; Max health
+    dw $09C6, $0000  ; Missiles
+    dw $09C8, $0000  ; Max missiles
+    dw $09CA, $0000  ; Supers
+    dw $09CC, $0000  ; Max supers
+    dw $09CE, $0000  ; Pbs
+    dw $09D0, $0000  ; Max pbs
+    dw $09D2, $0000  ; Currently selected item
+    dw $09D4, $0000  ; Max reserves
+    dw $09D6, $0000  ; Reserves
+    dw $0A1C, $0000  ; Samus position/state
+    dw $0A1E, $0000  ; More position/state
+    dw $0A68, $0000  ; Flash suit
+    dw $0A76, $0000  ; Hyper beam
+    dw $0AF6, $0080  ; Samus X
+    dw $0AFA, $0048  ; Samus Y
+    dw $0B3F, $0000  ; Blue suit
+    dw $D820, $0000  ; Events
+    dw $D822, $0000  ; Events
+    dw $D828, $0000  ; Bosses
+    dw $D82A, $0000  ; Bosses
+    dw $D82C, $0000  ; Bosses
+    dw $D82E, $0000  ; Bosses
+    dw $D870, $0000  ; Items
+    dw $D872, $0000  ; Items
+    dw $D874, $0000  ; Items
+    dw $D876, $0000  ; Items
+    dw $D878, $0000  ; Items
+    dw $D87A, $0000  ; Items
+    dw $D87C, $0000  ; Items
+    dw $D87E, $0000  ; Items
+    dw $D880, $0000  ; Items
+    dw $D882, $0000  ; Items
+    dw $D8B0, $0000  ; Doors
+    dw $D8B2, $0000  ; Doors
+    dw $D8B4, $0000  ; Doors
+    dw $D8B6, $0000  ; Doors
+    dw $D8B8, $0000  ; Doors
+    dw $D8BA, $0000  ; Doors
+    dw $D8BC, $0000  ; Doors
+    dw $D8BE, $0000  ; Doors
+    dw $D8C0, $0000  ; Doors
+    dw $D8C2, $0000  ; Doors
+    dw $D8C4, $0000  ; Doors
```

Comment:
I wrote a C# program to do all of the preset data work for me.  It's not pretty but it worked.